### ISSUE TITLE
keeweb-plugin watch using other port than standard

### ISSUE THREAD

**Issue by xieve** [keeweb/keeweb#817]
I tried to make a little theme to match keeweb with my xfce-theme-colors. When I did `keeweb-plugin watch my-theme` it gave this standard output:
```
KeeWeb plugin utils v0.1.8
This is your plugin URL for keeweb: https://127.0.0.1:8089
But first, open it in browser and click Proceed on unsafe website warning screen
Waiting for changes...
```
Though when I tried opening https://127.0.0.1:8089, nothing happened. (Firefox couldn't connect.) So I tried looking for open ports with `ss  -lntu` before starting `keeweb-plugin watch`:
```
udp   UNCONN     0      0                                                      127.0.0.1:53                                                                   0.0.0.0:*                  
udp   UNCONN     0      0                                                        0.0.0.0:68                                                                   0.0.0.0:*                  
udp   UNCONN     0      0                                                          [::1]:53                                                                      [::]:*                  
udp   UNCONN     0      0                                                              *:546                                                                        *:*                  
tcp   LISTEN     0      128                                                    127.0.0.1:53                                                                   0.0.0.0:*                  
tcp   LISTEN     0      128                                                        [::1]:53                                                                      [::]:*                  
tcp   LISTEN     0      128                                                            *:445                                                                        *:*                  
```
And after:
```
Netid State      Recv-Q Send-Q                                             Local Address:Port                                                            Peer Address:Port              
udp   UNCONN     0      0                                                      127.0.0.1:53                                                                   0.0.0.0:*                  
udp   UNCONN     0      0                                                        0.0.0.0:68                                                                   0.0.0.0:*                  
udp   UNCONN     0      0                                                          [::1]:53                                                                      [::]:*                  
udp   UNCONN     0      0                                                              *:546                                                                        *:*                  
tcp   LISTEN     0      128                                                    127.0.0.1:53                                                                   0.0.0.0:*                  
tcp   LISTEN     0      128                                                        [::1]:53                                                                      [::]:*                  
tcp   LISTEN     0      128                                                            *:445                                                                        *:*                  
tcp   LISTEN     0      128                                                            *:45889                                                                      *:*                  
```
I noticed the port 45889 was open, so I tried opening https://127.0.0.1:45889 __et voilà__, there it was, the security exception. Next time I started it it ran on port 40383. It works, but not quite well.
But when I tried importing it in keeweb, it said `Error loading plugin manifest`. I was able to access https://127.0.0.1:40383/manifest.json through Firefox though.
User-Agent:
```
KeeWeb v1.6.3 (undefined, 2017-12-22)
Environment: electron v1.6.15
User-Agent: Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) KeeWeb/1.6.3 Chrome/56.0.2924.87 Electron/1.6.15 Safari/537.36
```

**Comment by xieve** [comment]
I just tried importing it via `file://` URL and it seems to have found the manifest, but now it complains that the css property `--medium-color` is missing. I created this goddamn theme with the creator.

**Comment by xieve** [comment]
I'm sorry ':D

**Comment by antelle** [comment]
Fixed in `keeweb-plugin@0.1.9`, thanks!
